docs/quickstart/master/full.html

```diff
@@ -0,0 +1,953 @@
+</div>
+<div class="paragraph">
+<p>Use a terminal to run a Kafka producer and consumer on a local machine.</p>
+</div>
+<div class="ulist">
+<div class="title">Prerequisites</div>
+<ul>
+<li>
+<p>Strimzi is installed on the Kubernetes cluster</p>
+</li>
+<li>
+<p>Zookeeper and Kafka are running</p>
+</li>
+</ul>
+</div>
+<div class="olist arabic">
+<div class="title">Procedure</div>
+<ol class="arabic">
+<li>
+<p>Download the latest Kafka binaries and install Kafka on your local machine.</p>
+<div class="paragraph">
+<p><a href="http://kafka.apache.org/" target="_blank" rel="noopener">Apache Kafka download</a></p>
+</div>
+</li>
+<li>
+<p>Find the port of the bootstrap service:</p>
+<div class="listingblock">
+<div class="content">
+<pre class="highlightjs highlight nowrap"><code class="language-shell hljs" data-lang="shell">kubectl get service my-cluster-kafka-external-bootstrap -n my-kafka-project -o=jsonpath='{.spec.ports[0].nodePort}{"\n"}'</code></pre>
+</div>
+</div>
+</li>
+<li>
+<p>Find the IP address of the Minikube node:</p>
+<div class="listingblock">
+<div class="content">
+<pre class="highlightjs highlight nowrap"><code class="language-shell hljs" data-lang="shell">kubectl get nodes --output=jsonpath='{range .items[*]}{.status.addresses[?(@.type=="InternalIP")].address}{"\n"}{end}'</code></pre>
+</div>
+</div>
+</li>
+<li>
+<p>Open a terminal and start the Kafka console producer with the topic <code>my-topic</code>:</p>
+<div class="listingblock">
+<div class="content">
+<pre class="highlightjs highlight"><code class="language-shell hljs" data-lang="shell">bin/kafka-console-producer.sh --broker-list <em>&lt;node-address&gt;</em>:_&lt;node-port&gt;_ --topic my-topic</code></pre>
+</div>
+</div>
+</li>
+<li>
+<p>Type your message into the console where the producer is running.</p>
+</li>
+<li>
+<p>Press Enter to send the message.</p>
+</li>
+<li>
+<p>Open a new terminal tab or window and start the consumer to receive the messages:</p>
+<div class="listingblock">
+<div class="content">
+<pre class="highlightjs highlight"><code class="language-shell hljs" data-lang="shell">bin/kafka-console-consumer.sh --bootstrap-server <em>&lt;node-address&gt;</em>:_&lt;node-port&gt;_ --topic my-topic --from-beginning</code></pre>
+</div>
+</div>
+</li>
+<li>
+<p>Confirm that you see the incoming messages in the consumer console.</p>
+</li>
+<li>
+<p>Press Crtl+C to exit the Kafka console producer and consumer.</p>
+</li>
+</ol>
+</div>
+</div>
+</div>
+</div>
+</div>
+<div id="footer">
+<div id="footer-text">
+Last updated 2019-10-04 19:12:44 CEST
+</div>
+</div>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
+<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+<script>hljs.initHighlighting()</script>
+</body>
+</html>
```

Comment:
This file is generated? Does it need to be added to git?

Reply:
The TravisCI build normally generates these files in master build and pushes them to the website repo automatically. However, for the initial commit we have to add them manually here, merge this and then we can merge the PR strimzi/strimzi-kafka-operator#2053 which will be pushing it from the master builds automatically. If we merge this PR without them the website will have broken links until the other PR is merged and the Travi build there completes.